libs/langchain-community/src/vectorstores/cassandra.ts

```diff
@@ -49,7 +49,7 @@ export class CassandraStore extends VectorStore {
 
   private readonly table: CassandraTable;
 
-  private readonly idColumnAutoName = "_id";
+  private readonly idColumnAutoName = "id";
```

Comment:
This seems not backwards compatible?

Reply:
There's no harm since it's impossible that someone has a column named _id because it's not allowed in Cassandra. The feature wasn't working before